graphql/queries.tsx

```diff
@@ -109,11 +109,10 @@ query codeshare ($urlTitle: String!) {
 }`;
 
 export const CODE_SHARE_QUERY_LIST_ARTICLES = `
-query codeshare($offset: Int!; $tags: String!) {
+query codeshare($tags: String!) {
     CodeshareCollection(
-        limit: 10,
+        limit: 100000,
```

Comment:
Ask Daniel if we can pass a wildcard for "all" here.

Reply:
Apparently, there isn't.